### PR TITLE
Fixing Jammer Crystals

### PR DESCRIPTION
**Pull request by speed2CZ**
It was set to stop jamming only when killed, now it should work properly
even when reclaimed.
(Learning with Github part2: Lets hope I wont fuck up anything)

### CODE REVIEW

File: units/XSC9002/XSC9002_Script.lua [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    end,
+
+    OnKilled = function(self, instigator, type, excessDamageRatio)
+        if self.airChildUnit then self.airChildUnit:Destroy() end
+        if self.landChildUnit then self.landChildUnit:Destroy() end
+
+        SStructureUnit.OnKilled(self, instigator, type, excessDamageRatio)
+    end,
+
+    OnReclaimed = function(self, instigator, type)
```
Comment (CodingSquirrel) [inline units/XSC9002/XSC9002_Script.lua]: OnDestroy should handle both of these cases.

Comment (speed2CZ) [inline units/XSC9002/XSC9002_Script.lua]: well, it didn't

File: units/XSC9002/XSC9002_Script.lua [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            WaitSeconds(Random(7,13))
+
+            self.airChildUnit:Destroy()
+            self.airChildUnit = nil
+
+            #WaitSeconds(Random(2,7))
+        end
+    end,
+
+    OnDestroy = function(self, instigator, type, excessDamageRatio)
```
Comment (CodingSquirrel) [inline units/XSC9002/XSC9002_Script.lua]: OnDestroy only takes self as input. Same for where it's called on line 76

**Comment by ChrisKitching** [comment]
This should be one commit.

You can force push the branch you have in your PR to update the content here: no need to open new PRs if you bugger up the code (I do this *all the time* :P)

**Comment by speed2CZ** [comment]
I will need some advice how to do that, so far I work with trial/error method

**Pull request merged**